apps/generator-canonical-ds/src/component/templates/Component.tsx.ejs

```diff
@@ -0,0 +1,32 @@
+/* <%= generatorPackageName %> <%= generatorPackageVersion %> */
+import React from 'react'
+import type { <%= componentName %>Props } from './types.js'
+
+<% if (includeStyles) { %>
+import './<%= componentName %>.css'
+<% } %>
+  
+/**
+ * description of the <%= componentName %> component
+ * @returns {React.ReactElement} - Rendered <%= componentName %>
+ */
+export default function <%= componentName %>({
+  children,
+  className,
```

Comment:
Note from @goulinkh : We should consider including [clsx](https://www.npmjs.com/package/clsx) as an option for parsing classnames, as this is a quite popular and powerful addon for this use-case.

Reply:
Moved to #50